src/App.css

```diff
@@ -1,39 +1,71 @@
-.App {
-  text-align: center;
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+button {
+  background-color: inherit;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.calculator {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  max-width: 50vmin;
+  min-height: 50vmin;
+  margin-left: auto;
+  margin-right: auto;
 }
```

Comment:
![image](https://github.com/Diegogagan2587/math-magicians/assets/93039359/53a17b0f-e5c2-4b2b-9e00-7765a465e0ca)

- [ ] It would be nice if you make the zero on the right side of your screen to make your design look identical to the given design by microverse 👍